pkg/local_object_storage/metabase/graveyard.go

```diff
@@ -261,6 +262,52 @@ func (db *DB) DropGraves(tss []TombstonedObject) error {
 	})
 }
 
+// DropExpiredTSMarks run through the graveyard and drops tombstone marks with
+// tombstones whose expiration is _less_ than provided epoch.
+// Returns number of marks dropped.
+func (db *DB) DropExpiredTSMarks(epoch uint64) (int, error) {
+	db.modeMtx.RLock()
+	defer db.modeMtx.RUnlock()
+
+	if db.mode.NoMetabase() {
+		return -1, ErrDegradedMode
```

Comment:
dont see any sense in negative return, can be zero?

Reply:
just to highlight the error? changed but don't see a problem here